lib/groonga/groonga-log/parser.rb

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2017 Yasuhiro Horimoto <horimoto@clear-code.com>
+# Copyright (C) 2017 Kentaro Hayashi <hayashi@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+module Groonga
+  module GroongaLog
+    class Parser
+      EXTRACT_GROONGA_LOG_REGEXP =
```

Comment:
`FORMAT` or `PATTERN` is better.
`GROONGA_LOG` is redundant because we are in `GroongaLog` namespace.